config/initializers/0_application_config.rb

```diff
@@ -1,15 +1,17 @@
-ENVied.require(*ENV["ENVIED_GROUPS"] || Rails.groups)
-
 class ApplicationConfig
   URI_REGEXP = %r{(?<scheme>https?://)?(?<host>.+?)(?<port>:\d+)?$}.freeze
 
   def self.[](key)
-    ENVied.public_send(key)
+    if ENV.key?(key)
+      ENV[key]
+    else
+      Rails.logger.warn("Unset ENV variable: #{key}.")
+      nil
+    end
   end
 
   def self.app_domain_no_port
     app_domain = self["APP_DOMAIN"]
-    match = app_domain.match(URI_REGEXP)
-    match[:host]
+    app_domain&.match(URI_REGEXP)&.[](:host)
```

Comment:
**suggestion (non-blocking):** I think an early return would be easier to read than the `&.[](:host)` form:

```suggestion
    return unless app_domain
    
    app_domain.match(URI_REGEXP)[:host]
```